lib/galaxy/model/__init__.py

```diff
@@ -13,6 +13,7 @@
 import os
 import socket
 import time
+import pwd
```

Comment:
My bad! I think it should go before `import socket` to satisfy the alphabetic order.